### ISSUE TITLE
nvme bash completion

### ISSUE THREAD

**Issue by clavelc** [linux-nvme/nvme-cli#1258]
Hi,

Bash completion seems broken for the nvme command. Pressing **tab** after typing `nvme` prints an inline prompt: 

```
[cedric@modern15 ~]$ nvme [cedric@modern15 ~]$
```

See original post on [Arch forum](https://bbs.archlinux.org/viewtopic.php?pid=2005146#p2005146)

```
$ nvme --version
nvme version 1.16

$ uname -a
Linux modern15 5.15.4-arch1-1 #1 SMP PREEMPT Sun, 21 Nov 2021 21:34:33 +0000 x86_64 GNU/Linux
```

**Comment by sc108-lee** [comment]
it looks zsh(not bash completion) problem.
#1262 might be solved by this PR?

**Comment by clavelc** [comment]
Hi

Thanks for your answer. I don't know about zsh, I am using Bash.

**Comment by haromu** [comment]
I can confirm this error on my system.

```
$ nvme --version
nvme version 1.16

$ bash --version
GNU bash, version 5.1.12(1)-release (x86_64-pc-linux-gnu)
```

**Comment by igaw** [comment]
I just merged #1271 which should fix this problem. Closing the report, if is still breaking for you, please reopen it.
